avidemux_fr.ts

```diff
@@ -9943,12 +9943,12 @@ L&apos;édition en cours a été enregistrée et sera rechargée au prochain dé
     <message>
         <location line="+122"/>
         <source>Load video filters..</source>
-        <translation>Charger les filtres vidéo..</translation>
+        <translation>Charger les filtres vidéos..</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>Save video filters..</source>
-        <translation>Enregistrer les filtres vidéo..</translation>
+        <translation>Enregistrer les filtres vidéos..</translation>
```

Comment:
See the previous comment.

Reply:
It is true that the habit is to consider video as invariable as an adjective.
Since the 1990 reform, in the expression "video filters", the term "videos" has been considered as an adjective; it therefore agrees in number with the substantive. 

Reforms of French orthography, Wikipedia : https://en.wikipedia.org/wiki/Reforms_of_French_orthography#Rectifications_of_1990